src/index.html

```diff
@@ -39,12 +38,30 @@
 				top: 2rem;
 				width: 100%;
 			}
+
+			#btn {
+				display: flex;
+				justify-content: center;
+				background-color: white;
+				font-size: 2.5rem; 
+				position: fixed;
+				bottom: 2rem;
+				left: 2rem;
+				width: 3.3rem;
+				height: 3rem;
+				border-radius: 50%;
+				padding: 0.4rem;
+			}
 		</style>
 		<script defer src="./index.ts"></script>
 	</head>
 	<body>
 
 		<div id="points">0</div>
-
+		<button id="btn">
```

Comment:
This button could look better, but it's a minor issue

Reply:
I was not able to make a functional button on the canvas, might be because of threejs scene propetry.